### ISSUE TITLE
dl.freetutsdownload.net

### ISSUE THREAD

**Issue by chirag127** [AdguardTeam/AdguardFilters#96788]
**Issue URL (Annoyance)**:https://dl.freetutsdownload.net/redirect-to/?url=https%3A%2F%2Fbeaconschool-my.sharepoint.com%2F%3Au%3A%2Fg%2Fpersonal%2Fzcamacho_beaconschool_org%2FEQorlLR0tZRBuziWLDAU5v8B8q6Wq9ErAZlGwoLbjhlpaA

<details><summary>Screenshots:</summary>

![image](https://user-images.githubusercontent.com/76880977/136608096-ce4d9b7e-f8ee-4ea6-925a-11134c48849f.png)

</details><br/>

<details><summary>System configuration:</summary>

![image](https://user-images.githubusercontent.com/76880977/126797496-0434116b-fc94-42c9-8139-fba02a985cf2.png)
</details><br/>

**Comment by AdamWr** [comment]
Download button is already unblocked and it's even visible on your screenshot.
Duplicate of https://github.com/AdguardTeam/AdguardFilters/issues/94442#issuecomment-927677441
